File: trabalho 1/programas tarefa 1/java/micro/micro05.java
public class micro05{
    public static void main(String[] args){
        String nome, sexo;
        int x, h, m;
        h = 0;
        m = 0;
        for( x=1; x <= 5; x++){
            System.out.print("Digite o nome: ");
            nome = System.console().readLine();
            System.out.print("H - Homem ou M - Mulher:");
            sexo = System.console().readLine();
            if(sexo == "H"){
                h = h + 1;
            }else if(sexo == "M"){
                m = m + 1;
            }else{
                System.out.println("Sexo so pode ser H ou M!");
            }
        }
        System.out.println("Foram inseridos " + h + " Homens");
        System.out.println("Foram inseridos " + m + " Mulheres");
    }
}

File: trabalho 6/codigos/lua/micro/micro05.lua
function main() int
  string nome, sexo
  int x, h, m
  
  h = 0
  m = 0
  x = 1

  while x <= 5 do
    print("Digite o nome:")
    nome = io.read()
    print("H - Homem ou M - Mulher: ")
    sexo = io.read()
    if sexo == "H" then
      h = h + 1
    else 
      if sexo == "M" then
        m = m + 1
      else
        print("Sexo só pode ser H ou M!\n")
      end
    end
     x = x + 1
  end

  print("Foram inseridos", h, "Homens\n")
  print("Foram inseridos",m,"Mulheres\n")

end
